File: Common/Entity/src/main/java/it/ey/dto/ObiettivoPrevenzioneCorruzioneTrasparenzaDTO.java
package it.ey.dto;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import lombok.experimental.SuperBuilder;

import java.util.List;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
@SuperBuilder(toBuilder = true)
public class ObiettivoPrevenzioneCorruzioneTrasparenzaDTO  extends CampiTecniciDTO{

    private Long id;

    //Relazioni

    private Long idSezione23;
    private Long idOVP;
    private Long idStrategiaOVP;
    private Long idObbiettivoPerformance;


    private String codice;
    private String denominazione;
    private String descrizione;



     private List<ObiettivoPrevenzioneCorruzioneTrasparenzaIndicatoriDTO> indicatori;


}

File: dfp-piao-fe/src/app/shared/models/classes/obiettivo-prevenzione-corruzione-trasparenza-dto.ts
import { ObiettivoPrevenzioneCorruzioneTrasparenzaIndicatoriDTO } from './obiettivo-prevenzione-corruzione-trasparenza-indicatori-dto';

export class ObiettivoPrevenzioneCorruzioneTrasparenzaDTO {
  id?: number;
  idSezione23?: number;
  idOVP?: number;
  idStrategiaOVP?: number;
  idObbiettivoPerformance?: number;
  codice?: string;
  denominazione?: string;
  descrizione?: string;
  indicatori?: ObiettivoPrevenzioneCorruzioneTrasparenzaIndicatoriDTO[];
}
